src/Qwiq.Core/IQueryDefinition.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Microsoft.Qwiq
+{
+    public interface IQueryDefinition : IIdentifiable<Guid>, IEquatable<IQueryDefinition>
+    {
+        string Name { get; }
+        string QueryText { get; }
```

Comment:
WIQL?

Reply:
Sure, I was following the soap naming convention, which is QueryText, but Wiql is probably more clear.